airgun/views/dashboard.py

```diff
@@ -99,6 +100,11 @@ def search(self, query):
         self.searchbox.search(query)
         return self.read()
 
+    @View.nested
+    class DiscoveredHosts(View):
+        ROOT = ".//li[@data-name='Discovered Hosts']"
+        hosts = Table('.//table')
+
```

Comment:
@vijay8451 thanks for adding airgun PR :+1: 

Two Queries : 
1. Can you please also add the widget header as well?  It is a link containing the number of hosts ( case for assertion). 
2. There is also navigation from widget to details page, Do you think we need to add that flow as user will be doing that mostly ? 

![download](https://user-images.githubusercontent.com/3190629/55316152-6ac51900-548b-11e9-9951-c2a46add657e.png)

Reply:
@omkarkhatavkar thanks ..  
for queries , 1) Added widget in last commit 2) not looks that much usable one.